win/WIN_ToggleButton.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include "WIN_UIelement.h"
+#include <SDL.h>
+#include <vector>
+#include <memory>
+
+namespace win
+{
+	class ToggleButton;
+	typedef void (*ToggleActionFunction)(ToggleButton* button);
+	class ButtonGroup;
+	
+	class ToggleButton final :
+		public UIelement
+	{
+	public:
+		ToggleActionFunction action;
+		enum states { off, on };
```

Comment:
Coding Standard requires enum to be `State`. It should be similar to class name. Could also be `enum class`
**MUST**

Reply:
done